mu-plugins/global-fonts/index.php

```diff
@@ -54,22 +54,25 @@ function relative_to_absolute_urls( $editor_settings ) {
 /**
  * Specify a font to be preloaded.
  *
- * @param array|string $font_faces The font(s) to preload.
+ * @param string $fonts The font(s) to preload.
+ * @param string $subsets The subset(s) to preload.
  * @return bool If the font will be preloaded.
  */
-function preload_font( $font_faces ) {
+function preload_font( $fonts, $subsets ) {
 	$style = wp_styles()->query( 'wporg-global-fonts' );
-	if ( ! $style ) {
+	if ( ! $style || empty( $fonts ) || empty( $subsets ) ) {
 		return false;
 	}
 
-	if ( ! is_array( $font_faces ) ) {
-		$font_faces = [ $font_faces ];
-	}
+	$fonts = explode( ',', $fonts );
+	$subsets = explode( ',', $subsets );
 
 	$preload = $style->extra['preload'] ?? [];
-	$preload = array_merge( $preload, $font_faces );
-	$preload = array_unique( $preload );
+	$new_preload = [
+		'fonts' => array_unique( $fonts ),
+		'subsets' => array_unique( $subsets ),
```

Comment:
`array_unique` will be removed [here](https://github.com/WordPress/wporg-mu-plugins/pull/298/files#diff-3e55f21b4a7e4e1adfa13b4471290f4cce80b2f75fe9664fa35213977f7fb49cR72-R73). Check the link for more details.